docs/_posts/2023-11-07-tech-law-and-policy-year-in-review.md

```diff
@@ -0,0 +1,179 @@
+
+**The state-level funds and program implementation will be managed by the Arizona Commerce Authority.**
+
+**In the same vein, it’s worth noting Arizona businesses in rural areas may qualify for the U.S. Department of Agriculture’s [ReConnect](https://www.usda.gov/reconnect) Grand and Loan program.**
+
+**Arizona was awarded [$17.1 million of federal funding](https://www.rd.usda.gov/newsroom/news-release/usda-invests-171-million-bring-high-speed-internet-access-communities-rural-arizona) from the program in October last year. Individuals are not eligible for the ReConnect program, which is specifically meant to bring high-speed internet access to rural communities.**
+
+## How To Talk to the Government
+
+
+### State Legislature
+
+
+
+1. Register for the state Legislature’s [Request to Speak](https://apps.azleg.gov/Account/SignOn) system. This allows you to indicate approval or disapproval for bills with a thumbs up or thumbs down or, alternately, submit a written comment. You can also request to speak in person.
+   You’ll also want to bookmark [this list of acronyms](https://www.azleg.gov/faq/abbreviations/) if you’re new to the system. \
+
+2. Go down to the Capitol in person and complete registration at one of the kiosks. The kiosk is located in the State Senate, House of Representatives buildings at:
+
+1700 W. Washington St \
+Phoenix, AZ 85007
+
+Is having to visit a government building during the week during business hours a ridiculous obstacle to participating in the public policy process? Yes, yes it is.
+
+
+### Phoenix City Council
+
+Speaking at or submitting a comment for a Phoenix City Council meeting is also a bit cumbersome.
+
+**You must register online to comment at least two hours prior to the formal meeting, inconveniently held at 2:30 p.m. on a Wednesday.** Also, the Phoenix City Council hashes things out during its Policy Meetings, which are held separately, on a different day, from their Formal Meetings The Formal Meetings are where all the things figured out at the Policy Meeting get voted on and adopted so if you really want to follow or rabble rouse on a specific issue, it is often best to attend both.
+
+
+
+1. Go to this web page:[ https://www.phoenix.gov/cityclerk/publicmeetings/city-council-meetings \
+   ](https://www.phoenix.gov/cityclerk/publicmeetings/city-council-meetings)
+2. Scroll down a bit. Locate the meeting you’re interested in.
+3. Click the Comment icon to request to speak in person at the meeting or the keyboard icon to submit a written comment.
+
+
+**Comment in person:**
+
+a. Click on the dialogue bubble icon.
+
+b. Find the agenda item you want to comment on and fill out the form.
+
+c. **Arrive at the Chambers at least one hour early.** Meetings are held at the uber-convenient time of 2:30 p.m. on Wednesdays. Because who doesn’t have the luxury of taking time off work to be a civic participant?
+
+d. Go to the podium when called and speak your peace within two- to three-minutes.
```

Comment:
This looks like an ordered list and should use `#`s.

Reply:
Preserving Christina's content as-is.